client/components/marketing-survey/cancel-jetpack-form/index.tsx

```diff
@@ -60,17 +56,8 @@ const CancelJetpackForm: React.FC< Props > = ( {
 			return steps.CANCEL_CONFIRM_STEP;
 		}
 
-		// In these cases, the subscription is getting removed.
-		// Show the benefits step first.
-		if (
-			flowType === CANCEL_FLOW_TYPE.REMOVE ||
-			flowType === CANCEL_FLOW_TYPE.CANCEL_WITH_REFUND
-		) {
-			return steps.FEATURES_LOST_STEP;
-		}
-
 		return steps.CANCELLATION_REASON_STEP;
```

Comment:
Removing this condition is causing both the list of features lost and the cancellation survey to show when the flow type is "remove" when we only want to show the features lost for the removal flow.

I tested this with a purchase with a payment method that was past the refund window.
- On cancel I got: Survey
- On remove I got: Survey, list of features lost

IIRC the intent was to show the cancellation survey only when the subscription was cancelled and then show the benefits lost only before the subscription was removed to warn the user of the features that would be disabled.

Reply:
Yeah, nice catch!

The list of features lost was the default case, which was hit due to a step being unavailable. I restored the deleted code and added a few ifs so we now see it when the subscription is outside of the refund period and about to be removed.

<img width="1211" alt="Screenshot 2025-06-27 at 11 01 04" src="https://github.com/user-attachments/assets/6acea1ff-8cee-469e-b6a9-752ce9f0c101" />